runner_benchmark/taskset.py

```diff
@@ -32,8 +32,8 @@ def start(self):
         self.replay_requests()
 
     def replay_requests(self):
-        user_wait_time_min = int(os.getenv('USER_WAIT_TIME_MIN_SECONDS', 0))
-        user_wait_time_max = int(os.getenv('USER_WAIT_TIME_MAX_SECONDS', 0))
+        user_wait_time_min = int(os.getenv('USER_WAIT_TIME_MIN_SECONDS', 1))
```

Comment:
These defaults have been updated as the randrange usage later fails when both are set to the same value.